aws-xray-recorder-sdk-spring/build.gradle.kts

```diff
@@ -9,11 +9,11 @@ dependencies {
     api("org.aspectj:aspectjrt:1.8.11")
 
     // TODO(anuraaga): Remove most of these? Seems only Configurable annotation is used
```

Comment:
@anuraaga 's comment hints that these dependencies could be removed. If it is not a big effort, maybe this is a good time to do so?
Otherwise LGTM.

Reply:
Yeah Good callout. I think it's a small change relatively but in order to ensure nothing breaks and SDK works as expected we might need to do some additional testing so I have scoped that work out of this PR if that makes sense.